### DISCUSSION TITLE
help for markers

### DISCUSSION THREAD

**Discussion by zillionare** [python-poetry/poetry#4911] [Q&A]
I want to achieve this:

1. specify special package for apple m1
2. use generic package for other.

how to do this?

currently, I'm using:

Anything wrong when I'm using markers?

```
Bottleneck = [
    {url = "https://pypi.jieyu.ai/simple/bottleneck/Bottleneck-1.4.0.dev0+117.gf2bc792.dirty-cp39-cp39-macosx_11_0_arm64.whl", markers = "platform_machine == 'arm64' and 'sys_platform == 'darwin'"},
    {version = "*", markers = "platform_system == 'linux'"}
]
```
but this only specify deps for apple m1 and linux, what about other macos, linux arm and windows?

**Comment by TomerFi** [comment]
I think you can use "extras".

https://python-poetry.org/docs/pyproject/#extras
